<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>performance_analysis</class>
 <widget class="QWidget" name="performance_analysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Performance Analysis</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/FaceAttend2.png</normaloff>assets/FaceAttend2.png</iconset>
  </property>
  <layout class="QGridLayout" name="grid_layout">
   <item row="13" column="1">
    <widget class="QLabel" name="label_spacer_bottom">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_spacer_top">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <layout class="QFormLayout" name="form_layout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radio_enrollment">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_edit_enrollment">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Enrollment Number</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="radio_semester">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="combo_box_semester">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="combo_box_roll">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_placeholder_2">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Or, choose Semester and Roll Number:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_spacer_middle">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="15" column="0" colspan="3" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="button_update">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>assets/tick.png</normaloff>assets/tick.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_spacer_beneath">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <spacer name="horizontal_spacer_right">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="button_back">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>assets/return.png</normaloff>assets/return.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_performance_analysis">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#888a85;&quot;&gt;Performance Analysis&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <spacer name="horizontal_spacer_left">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QLabel" name="label_graph_icon">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>assets/analytics.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_placeholder_1">
     <property name="text">
      <string>You can check the performance of any student by their Enrollment Number:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>button_update</tabstop>
 </tabstops>
 <resources>
  <include location="FaceAttend.qrc"/>
 </resources>
 <connections/>
</ui>
